CHANGELOG.md

```diff
@@ -3,7 +3,9 @@ CHANGELOG
 
 ## HEAD (Unreleased)
 
-(None)
+- Normalize parameter names to singular names similar to properties
```

Comment:
Do we need to callout more explicitly that this is a breaking change? Wasn't clear on what we were planning to do on this front while we are in preview.

Reply:
We treated it light on azure native preview but maybe it's time to start doing so on google native